.dockerignore

```diff
@@ -1,2 +1,2 @@
-.*
```

Comment:
This was stopping the `.git` directory from being copied in which is needed to correctly resolve the version number. To avoid this blowing up the container size I've switched the `Dockerfile` to do a multi-stage build where we build the distributions and then only copy those over to the final container image.